packages/adapter/addon/json-api.ts

```diff
@@ -8,6 +8,11 @@ import { pluralize } from 'ember-inflector';
 import { serializeIntoHash } from './-private';
 import RESTAdapter from './rest';
 
+type Dict<T> = import('@ember-data/store/-private/ts-interfaces/utils').Dict<T>;
```

Comment:
Do we need <T>

Reply:
Yep.  The T in `type Dict<T>` is defining the generic and T in `.Dict<T>` is passing in that generic as an argument.